privacyidea/lib/user.py

```diff
@@ -112,6 +113,10 @@ def _get_user_from_userstore(self):
                 raise UserError("The resolver '{0!s}' does not exist!".format(
                     self.resolver))
             self.uid = y.getUserId(self.login)
+            if y.support_multiple_loginnames:
+                # In case the used loginname was not the primary loginname!
+                self.used_login = self.login
+                self.login = y.getUsername(self.uid)
```

Comment:
Currently, if ``y.support_multiple_loginnames == False``, ``self.used_login`` will be ``""``. But in order to have ``_log_used_user`` return the correct text, shouldn't it ``self.login``?

Reply:
As always you are right!

Reply:
done in 69b83c31

Reply:
:+1: